keps/4136-admission-fair-sharing/README.md

```diff
@@ -190,6 +191,22 @@ within AdmissionScope and then other mechanisms are applied as usual.
 workloads based on the LocalQueue resource usage when considering workloads within the same ClusterQueue. 
 Workloads from different ClusterQueues are not compared against each other using the resource usage dimension.
 
+### Entry penalty
+Because of the interval between updating FairSharingStatus the mechanism can be exploited. We may end up in a situation where one tenant
+can submit thousands of jobs which will be prioritized because they had slightly lower FairSharingStatus. 
+
+E.g. Let's assume:
+```
+Tenant's A FairSharingStatus: 0.1 CPU
+Tenant's B FairSharingStatus: 0.05 CPU
+usageSamplingInterval: 5mins
+```
+
+In the worst case scenario all of the jobs submitted by Tenant B will be prioritized and could be admitted for the next 5mins.
+Even if Tenant B submitted a job that consumed 1000 CPUs, consecutive jobs would still be prioritized because of the delay in updating the status.
```

Comment:
```suggestion
Here, all Jobs submitted by Tenant B  within the next 5min get scheduled.
Even if Tenant B submitted a job that consumed 1000 CPUs, consecutive jobs would still be prioritized because of the delay in updating the status.
```